content/tracing/setup/javascript.md

```diff
@@ -0,0 +1,163 @@
+
+```js
+const tracer = require('dd-trace').init()
+const opentracing = require('opentracing')
+
+opentracing.initGlobalTracer(tracer)
+```
+
+**NOTE: The tracer returned by `opentracing.globalTracer()` only
+contains OpenTracing specific methods.**
+
+**NOTE: When using OpenTracing, context propagation is not handled
+automatically.**
+
+## Integrations
+
+APM provides out-of-the-box instrumentation for many popular frameworks and libraries by using a plugin system. By default all built-in plugins are enabled. This behavior can be changed by setting the `plugins` option to `false` in the [configuration options](#configuration-options).
+
+Built-in plugins can be enabled by name and configured individually:
+
+```js
+const tracer = require('dd-trace').init({ plugins: false })
+
+// enable express integration
+tracer.use('express')
+
+// enable and configure postgresql integration
+tracer.use('pg', {
+  service: 'pg-cluster'
+})
+```
+
+See [compatibility](#compatibility) for the list of supported integrations.
+
+### Compatibility
+
+The `dd-trace` library includes support for a number of modules.
+
+#### Web Frameworks
+
+___
+
+| Module                                             | Versions | Support Type |
+|----------------------------------------------------|----------|--------------|
+| [express](https://expressjs.com/)                  | 4.x      | Experimental |
+| [hapi](https://hapijs.com/)                        |          | Coming Soon  |
+| [koa](https://koajs.com/)                          |          | Coming Soon  |
+| [restify](http://restify.com/)                     |          | Coming Soon  |
+
+#### Native Modules
+
+___
+
+| Module                                                        | Support Type |
+|---------------------------------------------------------------|--------------|
+| [http](https://nodejs.org/api/http.html)                      | Experimental |
+| [https](https://nodejs.org/api/https.html)                    | Experimental |
+
+#### Data Stores
+
+___
+
+| Module                                                             | Versions | Support Type |
+|--------------------------------------------------------------------|----------|--------------|
+| [elasticsearch](https://github.com/elastic/elasticsearch-js)       |          | Coming Soon  |
+| [ioredis](https://github.com/luin/ioredis)                         |          | Coming Soon  |
+| [mongodb-core](http://mongodb.github.io/node-mongodb-native/core/) |          | Coming Soon  |
+| [mysql](https://github.com/mysqljs/mysql)                          | 2.x      | Experimental |
+| [pg](https://node-postgres.com/)                                   | 6.x      | Experimental |
+| [redis](https://github.com/NodeRedis/node_redis)                   | >=2.6    | Experimental |
+
+### Distributed tracing
```

Comment:
Is this meant to be a sub-section of Integrations?

Reply:
Yes, but maybe it makes more sense to put it outside.

Reply:
Correct. Add it after the Opentracing section.